### ISSUE TITLE
Vedvarende oppdrag

### ISSUE THREAD

**Issue by aslakhol** [s0lvang/Karantenehjelpen#179]
**Hva ønsker du deg**
En av frivillighetssentralene har tatt kontakt og sagt at de trenger hjelp til å aktivisere brukerene sine. Det ledet meg til å tenke at vi kanskje kan åpne for en annen type oppdrag? 

Min første ide er vedvarende oppdrag. Med det mener jeg et oppdrag der ingen kan "ta oppdraget". Og dermed fortsetter å stå der helt til oppdragsgiveren tar det ned. 

Jeg er litt usikker på hvordan ting skal se ut når man lager oppdraget, kanskje det bare trenger å være en checkbox? 

Vi må vel også vise all kontaktinformasjonen hele tiden på disse.

**Comment by oleast** [comment]
Høres ut som en god idé å ha, hvordan er tankene rundt implementasjon?

Tenker i første omgang at det kan lagres i en annen 'collection' enn andre oppdrag, så man slipper å kjøre så mange conditionals på om det er denne eller denne typen oppdrag?
Skal vel allikevel gå an å hente inn begge typene som lister, sortere dem basert på en felles attributt og så rendre dem?

Hvordan er tanken på varighet? Skal det være helt til du selv tar den ned, eller kan du kanskje sette den opp til å vare til en viss dato (innenfor rimlige grenser)?

**Comment by plusk** [comment]
Har dere noen eksempler på hva et vedvarende oppdrag kan være? Er det da en i karantene som har gjengående bestillinger, f.eks. lufte hunden et par ganger om dagen? Eller er det hjelpere som skriver seg opp til å være tilgjengelige for oppdrag som kommer opp?

**Comment by aslakhol** [comment]
Ang. varighet så tenker jeg det er greit om man kan sette inn en dato den skal stå oppe til. Kanskje vi skal ha en makslengde også?

**Comment by aslakhol** [comment]
> Har dere noen eksempler på hva et vedvarende oppdrag kan være? Er det da en i karantene som har gjengående bestillinger, f.eks. lufte hunden et par ganger om dagen? Eller er det hjelpere som skriver seg opp til å være tilgjengelige for oppdrag som kommer opp?

Jeg tenker ikke at det passer for gjentagende oppdrag.  I hodet mitt så skal det dekke opp for sånne organisasjoner som frivillighetssentralen. Jeg tenker kanskje litt sånn som jobbannonsene vi har på onlinewebben?

**Comment by aslakhol** [comment]
Her er mailen jeg fikk, tror ikke de helt har skjønt hva vi gjør, men tenkte vi kunne lage noe som ville funke for de:

> Hei. Vi er 2 som jobber på jekken frivilligsentral.  Vi trenger hjelp til å komme i kontakt og aktivisere brukerne våre. Vi har masse ideer, men det stopper seg der. Eks felles matlaging på nett, dans, karaoke, spille spill.
> 
> Mange av våre brukere kommer seg ikke ut nå, og noen bor helt alene.
> 
> Er det noe dere kunne hjulpet oss med? Eks på mandag? Vi holder til på Voll gård på Moholt.
>

**Comment by Adrianht** [comment]
#153 kan kanskje gå litt under dette også?

**Comment by aslakhol** [comment]
Ja, de er nok overlappende! Jeg lar begge issuene stå oppe siden de har litt forskjellig vinkling, men de kan nok løses med samme løsning.
